CONTRIBUTING_PULLREQS.md

```diff
@@ -0,0 +1,23 @@
+### Requirements when making Code Changes (NOT watchlist/blacklist changes)
+
+When you are making pull requests to the code base for SmokeDetector, the
+Charcoal team needs he pull request to contain enough information for us to 
+quickly review things without requiring in-depth code analysis.
+
+**As such, a pull request that suggests code changes** (NOT blacklist or 
+watchlist changes) must have all or most of the following information:
+
+ - What does your pull request change or introduce to the project?
+ - What is the justification for the inclusion of your Pull Request (or, what 
+   problem does it solve?)
+ - What does your code (if new) actually do?  Summarize what your new code 
+   does (either within the code via comments or with your Pull Request)
+   
+Pull requests that do not have ample information attached to them for 
+justification, etc. may be rejected or may be put on hold until we get more 
+information. Failure to provide ample information must 
```

Comment:
must?

Reply:
Force of habit to write this, we can revise this easy.

Reply:
Must what, though?

Reply:
gdi autocorrect.  This didn't copy and then when i retyped failed things.  Altered.